### DISCUSSION TITLE
v2.9.3

### DISCUSSION THREAD

**Discussion by iBotPeaches** [iBotPeaches/Apktool#3495] [Releases]
- [[#3492](https://github.com/iBotPeaches/Apktool/pull/3492)] Fix [#GHSA-vgwr-4w3p-xmjv](https://github.com/iBotPeaches/Apktool/security/advisories/GHSA-vgwr-4w3p-xmjv) (Arbitrary file writes on Windows). (Thanks Cl0udG0d)

**Full Changelog**: https://github.com/iBotPeaches/Apktool/compare/v2.9.3...v2.9.3
**Doc Site**: https://apktool.org/blog/apktool-2.9.3
**Release Blog**: https://connortumbleson.com/2024/01/20/apktool-v2-9-3-released/

<hr /><em>This discussion was created from the release <a href='https://github.com/iBotPeaches/Apktool/releases/tag/v2.9.3'>v2.9.3</a>.</em>

**Comment by vardinioboss** [comment]
how to install it 😭

**Reply by iBotPeaches** [reply]
https://apktool.org/docs/install
